apps/core/src/utils/stake/getTransactionAmountForTimelocked.ts

```diff
@@ -15,10 +15,10 @@ export function getTransactionAmountForTimelocked(
     const { isTimelockedStaking, isTimelockedUnstaking } = checkIfIsTimelockedStaking(events);
```

Comment:
We can pass this info by prop and remove that function from here. In the places where `getTransactionAmountForTimelocked` is called, it is checked if the tx is timelocked or not, so `checkIfIsTimelockedStaking` is called twice